types/draft-convert/index.d.ts

```diff
@@ -0,0 +1,26 @@
+// Type definitions for draft-convert 1.4
+// Project: https://github.com/hubspot/draft-convert
+// Definitions by: guicapanema <https://github.com/guicapanema/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+export function convertFromHTML(...args: any[]): any;
+
+export function convertToHTML(...args: any[]): any;
+
+export function parseHTML(html: any): any;
+
+export namespace convertFromHTML {
```

Comment:
Did you use `dts-gen`? This looks like a bug; every function has an empty prototype by default, but that shouldn't need to be declared.